### PR TITLE
Upgrade to Maven Wrapper 3.2.0

### PR DESCRIPTION
**Pull request by jchrys**
Motivation:
Ensure up-to-date with Maven Wrapper

Modification:
Use Maven Wrapper 3.2.0

Result:
Up-to-date

### CODE REVIEW

**Review by JohnNiang — COMMENTED** [review]
Hi @jchrys , I suggest using `mvn wrapper:wrapper -Dmaven=3.2.0` to upgrade Maven Wrapper. Please refer to <https://maven.apache.org/wrapper/#using-a-different-version-of-maven>.

```bash
git status
HEAD detached at upstream/trunk
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   .mvn/wrapper/maven-wrapper.properties
	modified:   mvnw
	modified:   mvnw.cmd
```

**Comment by jchrys** [comment]
@JohnNiang Thanks a lot!

**Comment by JohnNiang** [comment]
As far as I know from <https://github.com/asyncer-io/r2dbc-mysql/pull/133>, we should upgrade Maven to 3.9.3. So we might need to execute `mvn wrapper:wrapper -Dmaven=3.9.3`

**Comment by jchrys** [comment]
I just tried to execute `mvn wrapper:wrapper -Dmaven=3.9.3`. but it seems OK. I couldn't find diffs.

**Comment by JohnNiang** [comment]
> I just tried to execute `mvn wrapper:wrapper -Dmaven=3.9.3`. but it seems OK. I couldn't find diffs.

Please refer to https://github.com/asyncer-io/r2dbc-mysql/compare/trunk...JohnNiang:chore/upgrade-maven-wrapper?expand=1

**Comment by jchrys** [comment]
I see. Thanks a lot. It seems that we've been used non-official version(or EOL). Would you mind if I create PR using yours?

**Comment by JohnNiang** [comment]
> I see. Thanks a lot. It seems that we've been used non-official version(or EOL). Would you mind if I create PR using yours?

Of course! Feel free to use the information and resources I provided to create the PR. If you have any questions or need further assistance during the process, feel free to ask.

File: mvnw.cmd
Comment (JohnNiang) [inline mvnw.cmd:1]: Please note that the line terminators of Windows batch file should be `CRLF`:

```bash
file mvnw.cmd
mvnw.cmd: DOS batch file text, ASCII text, with `CRLF` line terminators
```

But I found the line terminators of `mvnw.cmd` in the PR was not `CRLF`, please see the result below:

```bash
file mvnw.cmd
mvnw.cmd: DOS batch file text, ASCII text
```

**Comment by jchrys** [comment]
:D I just applied it. @JohnNiang PTAL

**Review by JohnNiang — APPROVED** [review]
LGTM

**Pull request merged**